I-D_in_xml/draft-ietf-ccamp-optical-impairment-topology-yang.xml

```diff
@@ -347,9 +347,16 @@
   </figure>
   
   <t>
-   BA (on the left side WDM-TE-node) is the engress Amplifier and PA
-   (on the right side WDM-TE-node is the ingress amplifier for the OMS
-   Media Channel Group (MCG) <xref target="Figure-2"/>.
+   According to <xref target="G.807"/>, an
+   OMS Media Channel Group (MCG) represents "a unidirectional 
+   point-to-point management/control abstraction that represents a set 
+   of one or more media channels that are co-routed. A media channel 
+   group (MCG) is bounded by a pair of media ports." 
```

Comment:
It seems to me that this is the generic definition of the MCG rather than the specific definition of OMS MCG, which is a special type of MCG defined by G.807

For example see the specific definition of OTS MCG below